packages/shared/locales/en.json

```diff
@@ -1336,6 +1336,18 @@
         "wallet": "Wallet",
         "staking": "Staking"
     },
+    "network": {
+        "iota": {
+            "mainnet": "IOTA Mainnet",
+            "devnet": "IOTA Devnet",
+            "private-net": "IOTA Private-net"
+        },
+        "shimmer": {
+            "mainnet": "Shimmer Mainnet",
+            "devnet": "Shimmer Devnet",
+            "private-net": "Shimmer Private-net"
+        }
+    },
```

Comment:
Not sure we should have these as locales given they are nouns, and sometimes come from the node them selves.

Reply:
Ahhh wasn't sure if we printed IOTA and Shimmer differently in non-latin alphabets.